test/access-token.test.js

```diff
@@ -388,7 +388,7 @@ describe('AccessToken', function() {
       var Token = givenLocalTokenModel();
 
       // Overwrite User settings - enable eternal tokens
-      Token.app.models.User.settings.allowEternalTokens = true;
+      app.registry.getModelByType(loopback.User).settings.allowEternalTokens = true;
```

Comment:
`getModelByType('User')`

I am surprised this code actually works for you, considering that `givenLocalTokenModel` is using a local per-app registry.